app/src/main/java/org/oppia/android/app/player/state/StateViewModel.kt

```diff
@@ -66,6 +77,12 @@ class StateViewModel @Inject constructor() : ObservableViewModel() {
     }
   }
 
+  private fun getRawUserAnswerWithError(
```

Comment:
Is this helper actually needed? It doesn't seem to be adding much. Could we just inline it?